Firestore/Source/Remote/FSTRemoteEvent.mm

```diff
@@ -330,6 +342,10 @@ - (void)synthesizeDeleteForLimboTargetChange:(FSTTargetChange *)targetChange
   return _documentUpdates;
 }
 
+- (const firebase::firestore::model::SnapshotVersion &)snapshotVersion {
```

Comment:
In the .mm you can using a `using` declaration to shorten this and other usages up.

Reply:
done